devicetypes/qubino/qubino-flush-shutter.src/qubino-flush-shutter.groovy

```diff
@@ -0,0 +1,501 @@
+/**
+ *	Copyright 2020 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+import java.lang.Math
```

Comment:
you shouldn't have to import this I don't think

Reply:
I know that Groovy has max/min included from the scratch, but without this import I've received exceptions

Reply:
I think you can just reference `Math.round` without an explicit import.

Reply:
At least, that's the way it's done on several other DTHs

Reply:
can you give this a shot?

Reply:
It worked.